package.json

```diff
@@ -285,10 +285,12 @@
     "raven-js": "3.27.2",
     "razzle": "3.0.0",
     "razzle-plugin-bundle-analyzer": "1.1.3",
+    "rc-time-picker": "4.0.0-alpha.2",
```

Comment:
@nzambello any particular reason, you chose the alpha here instead of the stable 3.x release? http://react-component.github.io/time-picker/

Reply:
Yarn installed that. Switched to stable version.